src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -4569,7 +4569,7 @@ INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveBiDirection
 INSERT INTO tab_calculation_method_rates VALUES('DynamicSumFormulas','PARAM_SumOfActiveProcessRatesLumenToMucosa','P_#i','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('DiseaseStates','PARAM_TargetGFR','60 * 1E-2 * 1E-3 / 1.73','Flow per body surface area');
 INSERT INTO tab_calculation_method_rates VALUES('GFR_Calculation','PARAM_eGFR','GFR / BSA','Flow per body surface area');
-INSERT INTO tab_calculation_method_rates VALUES('DiseaseStates','One_Rate','1','Fraction');
+INSERT INTO tab_calculation_method_rates VALUES('DiseaseStates','One_Rate','1','Dimensionless');

```

Comment:
PArameters are in fact instantiated in code and the dim was already Dinensionsless.